csharp-basics/exercises/Loops/Loops/NumberSquare/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace NumberSquare
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int min, max;
+
+            Console.WriteLine("Min?");
+            min = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Max?");
+            max = Convert.ToInt32(Console.ReadLine());
+
+            for (int i = min; i <= max; i++)
+            {
+                int valueToMax = i;
```

Comment:
Kāpēc tev ir šis jādara un nevari izmantot i un min?

Reply:
Jo es vinus mainu un lai izvairītos no riska mainīt, tās vertības lieku jaunus mainīgos.

Reply:
Pamanīju tagad